### PR TITLE
Fix/timer bug

### PR DESCRIPTION
**Pull request by owenhallgren**
## Is this a fix or a feature?
Fix
## What is the change?
N/A
## What does it fix?
This fixes the timer going off late after click
This also fixes multiple timers going off at once
## Where should the reviewer start?
Review the method countdown
## How should this be tested?
The reviewer should try to start multiple timers

### CODE REVIEW

**Review by Jacksonmcguire — APPROVED** [review]

File: activity.js [hunk 1]
```diff
@@ -6,16 +6,27 @@ class Activity {
...
+    time--;
+    var minutes = String(Math.trunc(time / 60)).padStart(2, 0)
+    var seconds = String(Math.trunc(time % 60)).padStart(2, 0)
+    timer.textContent = `${minutes}:${seconds}`;
     var decreaseTime = setInterval(function() {
+      time--;
       var minutes = String(Math.trunc(time / 60)).padStart(2, 0)
       var seconds = String(Math.trunc(time % 60)).padStart(2, 0)
       timer.textContent = `${minutes}:${seconds}`;
       time--;
```
Comment (elizhahn) [inline activity.js]: Maybe we can refactor this down

File: activity.js [hunk 2]
```diff
@@ -6,16 +6,27 @@ class Activity {
...
   };
   countdown(){
+    if(this.timerRunning == true){
+      return;
+    }
+    this.timerRunning = true;
     var time = Number(this.minutes * 60) + Number(this.seconds)
+    time--;
+    var minutes = String(Math.trunc(time / 60)).padStart(2, 0)
+    var seconds = String(Math.trunc(time % 60)).padStart(2, 0)
```
Comment (Jacksonmcguire) [inline activity.js]: don't forget to add semicolons

File: activity.js [hunk 3]
```diff
@@ -6,16 +6,27 @@ class Activity {
...
+    timer.textContent = `${minutes}:${seconds}`;
     var decreaseTime = setInterval(function() {
+      time--;
```
Comment (elizhahn) [inline activity.js:22]: This is decrementing each time by 2 seconds

File: activity.js (hunk 2 above)
Comment (owenhallgren) [reply activity.js]: On that!

File: activity.js (hunk 1 above)
Comment (owenhallgren) [reply activity.js]: I'll look into adding a function for setting the variables

File: activity.js (hunk 3 above)
Comment (owenhallgren) [reply activity.js:22]: Just added

**Review by Jacksonmcguire — APPROVED** [review]

**Review by elizhahn — APPROVED** [review]

**Pull request merged**